masschallenge/application-allocator
# [AC-5619] Add React

To test:
- Run `make run-server`

Should startup a webserver and give you a local URL you can check out.  The local URL should just say "Hello, world".

- Run `make test`

Should successfully run a single test.
